### ISSUE TITLE
Nodeshift keeps adding it's own DeploymentConfig to items

### ISSUE THREAD

**Issue by relief-melone** [nodeshift/nodeshift#271]
I am not compeltely sure if this is an issue but have not been able to resolve this.

I keep a deployment.yml route.yml and svc.yml in my .nodeshift folder.

However when I deploy (using nodeshift.deploy in js-file) in my tmp/nodeshift/resource folder nodeshift keeps adding it's own DeploymentConfig at the end of items. This is also the one that OpenShift then seems to use to deploy. I also tried different filenames, but nothing worked so far. Am I doing something wrong or is this an issue?

**Comment by lholmquist** [comment]
@relief-melone  in your deployment.yml, do you specify the "Kind" property?  if you could, could you share maybe the first like 5-10 lines of that yaml file.  

I think i might know the issue, but i need to see a little bit of that deployment yaml file first

**Comment by relief-melone** [comment]
Hey. Sorry for the late reply. I thought about a typo in the kind too at first but all looks fine here I guess

```
apiVersion: v1
kind: DeploymentConfig
metadata:
  name: myapp
  namespace: app-sandbox
  labels:
```

**Comment by lholmquist** [comment]
@relief-melone i'm pretty sure this is the issue.  In this file, https://github.com/nodeshift/nodeshift/blob/master/lib/resource-enrichers/deployment-config-enricher.js#L50,  nodeshift checks to see if you have specified a deployment yaml,  but it only looks for the "kind: Deployment", not "kind: DeploymentConfig".  It should look for both.  

Should be an easy fix.

If you don't want to wait until i push a new version, you can just change "DeploymentConfig" to "Deployment" in your yaml file

**Comment by lholmquist** [comment]
@relief-melone just released 2.0.1 on npm with the fix
